lib/web_console/templates/console.js.erb

```diff
@@ -80,13 +80,34 @@ REPLConsole.prototype.commandHandle = function(line, callback) {
     return status >= 200 && status < 300 || status === 304;
   }
 
+  function parseJSON(text) {
+    try {
+      return JSON.parse(text);
+    } catch (e) {
+      return null;
+    }
+  }
+
+  function getErrorText(xhr) {
+    if (!xhr.status) {
+      return "Oops! Failed to connect to Web Console middleware.\n" +
```

Comment:
It may be a good idea to collect all the messages we have and put them in config/locale/en.yml or something like this. It will open the door for internationalization and will get rid of those hard-coded messages from the code-base. Just some food for thought.